assets/src/edit-story/components/keyboardShortcutsMenu/popup.js

```diff
@@ -48,7 +48,7 @@ const transitionStyles = {
 const Controller = styled.div`
   position: absolute;
   top: -12px;
-  right: 0;
+  left: 0;
   opacity: 0;
   z-index: 1;
   transform: translateX(20px);
```

Comment:
Should we make these `-20` for here and the exitStyles so that it matches directionally animation wise with the help center?
```suggestion
  transform: translateX(-20px);
```

Reply:
Oh yeah, forgot to flip these! 🤦‍♂️

Reply:
Fixed in 5bf4a9445.